colony/commands/bp.py

```diff
@@ -41,7 +41,11 @@ def do_validate(self):
         if commit and branch is None:
             raise DocoptExit("Since a commit was specified, a branch parameter is also required")
 
-        repo, working_branch, temp_working_branch, stashed_flag = figure_out_branches(branch, blueprint_name)
+        repo, working_branch, temp_working_branch, stashed_flag, success = figure_out_branches(branch, blueprint_name)
+
+        if not success:
```

Comment:
Because the behavior would become hard to predict.
Its hard enough explaining our branching logic (local changes do one thing, none do another) without also introducing a "fallback" behavior.
They can still use the --branch flag to check anything on the remote branch